### DISCUSSION TITLE
MBZ bug in EDE code?

### DISCUSSION THREAD

**Discussion by mdavids** [PowerDNS/pdns#14547] [Q&A] [answered]
I added [some comments](https://github.com/PowerDNS/pdns/pull/13473#issuecomment-2271061159) to https://github.com/PowerDNS/pdns/pull/13473, because I suspect a tiny bug that caused the OPT Z-flag to be filled instead of the DO-flag.

<img width="555" alt="bugornot" src="https://github.com/user-attachments/assets/b149601e-d98c-4bfa-9eda-0bf7c262bd36">

What do you think?

**Comment by Habbie** [comment, marked as answer]
oops. I missed `+do` in your earlier notes. Bug reproduced!

**Reply by Habbie** [reply]
Sorry to drive you around like this, but now that it's reproduced, a fresh -Issue- would be best :)

**Reply by mdavids** [reply]
Sorry for causing the confusion and for so many edit's. I also missed the `+do` myself, because it is in my `.digrc` ... 🤦🏼‍♂️ 
(not the first time that 'handy' file bites me). I'll create an issue.

**Reply by Habbie** [reply]
ha, yes, digrc strikes often. I did read back to see if I missed it or if it was, indeed, implied ;)
